src/platform/utilities/feature-toggles/featureFlagNames.js

```diff
@@ -1,7 +1,6 @@
 export default Object.freeze({
   facilityLocatorShowCommunityCares: 'facilityLocatorShowCommunityCares',
   profileShowReceiveTextNotifications: 'profileShowReceiveTextNotifications',
-  profileSchemaForms: 'profileSchemaForms',
```

Comment:
this flag is also deleted on the backend here https://github.com/department-of-veterans-affairs/vets-api/pull/4038